csharp.test/PhysicalValueGetter.cs

```diff
@@ -10,8 +10,7 @@ public PhysicalValueGetter(long numValues)
             _numValues = numValues;
         }
 
-        public (Array values, short[] definitionLevels, short[] repetitionLevels) OnColumnReader<TValue>(ColumnReader<TValue> columnReader) 
-            where TValue : unmanaged
+        public (Array values, short[] definitionLevels, short[] repetitionLevels) OnColumnReader<TValue>(ColumnReader<TValue> columnReader) where TValue : unmanaged
```

Comment:
Same remark as above. The previous style was more readable (`where` on a separate line).